### PR TITLE
Chat: rename option `errors` to `alerts`

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

File: packages/devextreme/js/ui/chat.d.ts [hunk 1]
```diff
@@ -100,7 +100,7 @@ export type User = {
  * @namespace DevExpress.ui.dxChat
  * @public
  */
-export type ChatError = {
+export type Alerts = {
```
Comment (vladaskorohodova) [inline packages/devextreme/js/ui/chat.d.ts]:
```suggestion
export type Alert = {
```

File: packages/devextreme/js/ui/chat.d.ts [hunk 2]
```diff
@@ -195,7 +195,7 @@ export interface dxChatOptions extends WidgetOptions<dxChat> {
      * @default undefined
      * @public
      */
-    errors?: Array<ChatError>;
+    alerts?: Array<Alerts>;
```
Comment (vladaskorohodova) [inline packages/devextreme/js/ui/chat.d.ts]:
```suggestion
    alerts?: Array<Alert>;
```

File: packages/devextreme/js/ui/chat_types.d.ts [hunk 3]
```diff
@@ -6,7 +6,7 @@ export {
   TypingStartEvent,
   TypingEndEvent,
   User,
-  ChatError,
+  Alerts,
```
Comment (vladaskorohodova) [inline packages/devextreme/js/ui/chat_types.d.ts]:
```suggestion
  Alert,
```

File: packages/devextreme/js/ui/chat.d.ts (hunk 2 above)
Comment (EugeniyKiyashko) [reply packages/devextreme/js/ui/chat.d.ts]: https://github.com/DevExpress/DevExtreme/pull/28270/commits/1266f1d1cc55dbba7024574dc36420a4c999f52b

File: packages/devextreme/js/ui/chat.d.ts (hunk 1 above)
Comment (EugeniyKiyashko) [reply packages/devextreme/js/ui/chat.d.ts]: https://github.com/DevExpress/DevExtreme/pull/28270/commits/1266f1d1cc55dbba7024574dc36420a4c999f52b

**Review by vladaskorohodova — DISMISSED** [review]

**Review by ksercs — DISMISSED** [review]

**Review by ksercs — DISMISSED** [review]

**Review by vladaskorohodova — DISMISSED** [review]

**Review by mpreyskurantov — DISMISSED** [review]

**Review by mpreyskurantov — APPROVED** [review]

**Review by vladaskorohodova — APPROVED** [review]

**Pull request merged**